packages/openneuro-components/src/range/TwoHandleRange.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Slider from 'rc-slider'
+import 'rc-slider/assets/index.css'
+import './range.scss'
+
+export interface TwoHandleRangeProps {
+  min: number
+  max: number
+  step: number
+  dots: boolean
+  pushable: number | boolean
+  defaultValue: [number, number]
+  marks: { number: string }
+  newvalue: [number, number]
+  setNewValue: [number, number]
+}
+const createSliderWithTooltip = Slider.createSliderWithTooltip
+const Range = createSliderWithTooltip(Slider.Range)
```

Comment:
This can probably be simplified to `const Range = Slider.createSliderWithTooltip(Slider.Range)`